src/app/components/header/header.component.ts

```diff
@@ -1,10 +1,27 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrl: './header.component.scss'
 })
-export class HeaderComponent {
-  title = 'Pokemon App';
+export class HeaderComponent implements OnInit {
+  isLoggedIn: boolean = false;
+
+  constructor(private router: Router) {}
+
+  ngOnInit(): void {
+    if (typeof sessionStorage !== 'undefined') {
+      this.isLoggedIn = sessionStorage.getItem('loggedInEmail') === 'true';
+    }
```

Comment:
if the value of something if true you can check
if(sessionStorage.getItem('loggedInEmail')){}